src/piidigger/filescan.py

```diff
@@ -48,7 +41,8 @@ def findDirsWorker(config: classes.Config,
         for d in config.getStartDirs():
             localQ.append(pathlib.Path(d))
             queues['dirsQ'].put(pathlib.Path(d))
-            totals['dirsFound'].value+=1
+            with totals['dirsFound'].get_lock():
```

Comment:
Found an instance of using a shared variable without a lock.  This was unlikely to cause any issues as findDirs will likely only be a single process, but still... fixed it just in case.